src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -2407,6 +2407,9 @@ void ColorControlServer::updateXYCommand(EndpointId endpoint)
     Color16uTransitionState * colorXTransitionState = getXTransitionStateByIndex(epIndex);
     Color16uTransitionState * colorYTransitionState = getYTransitionStateByIndex(epIndex);
 
+    VerifyOrReturn(colorXTransitionState != nullptr);
```

Comment:
would it make sense to log something? otherwise it would be very hard to debug a "updating XY silently fails" issue.